estate/__manifest__.py

```diff
@@ -0,0 +1,11 @@
+{
+    'name': 'RealEstate',
+    'version': '1.0.0',
```

Comment:
```suggestion
    'version': '1.0',
```
We use versions with 2 numbers.